.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+This document defines many guidelines that should be adhered to when developing 
+against Banshee. These guidelines will make the codebase more readable,
+extensible, and portable.
+
+Reporting Issues
+================
+
+[GNOME's bugzilla](https://bugzilla.gnome.org/buglist.cgi?quicksearch=product%3Abanshee)
+is the official issue tracker and contributors are encouraged to file bugs and
+feature requests there in addition to this repository.
```

Comment:
IMHO the length is still okay.
If you want to split it, you could consider putting the parts into the docs folder and then link to them.